### ISSUE TITLE
When the transaction is constructed, the balance cannot be matched.

### ISSUE THREAD

**Issue by peiyuyaoyaou** [mycelium-com/wallet-ios#46]
There is currently 0.0006BTC transfer 0.000048BTC fee: 0.0001
The original balance should have 0.00002, but the balance becomes 0.00 after the transfer. What is going on?
Printed the next data BTCTransactionOutput only built a 0.000048BTC transaction and did not build a zero change transaction with a balance of 0.00002. 

In addition, when the wallet has 0.0006BTC transfer 0.00004BTC fee: 0.0001
At this point, a 0.00004 and a 0.0001 change transaction will be constructed. This time the balance is normal.
In short, when the balance is large after the transaction, the transaction is normal. If the balance is small, it will be cleared directly. This makes me confused.
I hope to hear from you, thank you!！
